modules/rosa-sts-aws-requirements.adoc

```diff
@@ -24,7 +22,7 @@ Customers are encouraged, but not mandated, to deploy resources in a Virtual Pri
 [id="rosa-associating-account_{context}"]
 == Associating your AWS account
 
-To perform {product-title} (ROSA) cluster provisioning tasks, you must create `ocm-role` and `user-role` IAM resources in your AWS account and link them to your Red Hat organization.
+{product-title} (ROSA) cluster provisioning tasks require linked `ocm-role` and `user-role` {cluster-manager} IAM resources to your AWS account using your Amazon Resource Name (ARN).
```

Comment:
s/cluster provisioning/cluster-provisioning ?? (I had to read it a few times to understand.)

s/linked/linking ?? require the `ocm-role` and `user-role` {cluster-manager} IAM resources be linked to   (Seems like there needs to be another verb somewhere.)